lighthouse-core/audits/uses-rel-preconnect.js

```diff
@@ -28,6 +29,8 @@ const UIStrings = {
   description:
     'Consider adding preconnect or dns-prefetch resource hints to establish early ' +
     `connections to important third-party origins. [Learn more](https://developers.google.com/web/fundamentals/performance/resource-prioritization#preconnect).`,
+  /** A warning message that is shown when the user tried to follow the advice of the audit, but it's not working as expected. Forgetting to set the `crossorigin` HTML attribute on the link is a common mistake when adding preconnect links. */
+  crossoriginWarning: 'A preconnect link was found for "{securityOrigin}" but was not used.',
```

Comment:
@wardpeet and I discussed and he suggested removing the mention of crossorigin, I somewhat agree and given our other positions it makes some sense to keep it in the docs

> Now your warning message isn't super helpful. If I read it I would add crossorigin to the preconnect tag and I'll see the same message pop up. Sometimes crossorigin helps but the urls listed in the bugs do not get fixed by slapping crossorigin on it.

Reply:
- would it be terrible to do `"A preconnect <link> was..."` to make it clear what's being referenced?
- if not going to mention crossorigin, does the string var need a new name?
- `"was not used"` is kind of ambiguous. `"was not used by the browser"`? `"was not able to be used by the browser"`? followed? honored?
- Is there any more hint we can give about what went wrong?

Reply:
> would it be terrible

nope sounds good

> if not going to mention crossorigin, does the string var need a new name?

heh, yes, but hang on one sec

> is kind of ambiguous

`was not used by the browser` SGTM

> Is there any more hint we can give about what went wrong?

This is 99% a `crossorigin` problem IMO, since you and Paul both are on that side and that was my first instinct as well, I think it's worth going back :) sorry @wardpeet!